scpdt/plugin.py

```diff
@@ -159,6 +162,58 @@ def collect(self):
 
 def _get_parser():
     """
-    Return instance of DTParser with default configuration
+    Return instance of DTParser
+    """
+    return DTParser(config=dt_config)
+
+
+def _get_runner(checker, verbose, optionflags):
+    import doctest
+    """
+    Override function to return instance of DebugDTRunner
     """
-    return DTParser(config=dt_config)
\ No newline at end of file
+    class PytestDTRunner(DebugDTRunner):
+
+        def __init__(self, checker=None, verbose=None, optionflags=None, config=None, continue_on_failure=True):
+            super().__init__(checker=checker, verbose=verbose, optionflags=optionflags, config=config)
+            """
+            If `continue_on_failure is True, failures and exceptions are collected in the out stream. 
+            If it's False, the failure or exception is raised immediately, potentially stopping further 
+            execution of the particular doctest
+            """
+            self.continue_on_failure = continue_on_failure
```

Comment:
`config` argument is ignored, what is `self.config` below in `self.config.user_context_mgr`. This ties up to a question of where the global DTConfig lives.